src/Domain/Calculator/CalculatorInterface.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PhpFinance\DoubleEntry\Domain\Calculator;
+
+use Brick\Money\MoneyBag;
+use PhpFinance\DoubleEntry\Domain\Account\Account;
+
+interface CalculatorInterface
```

Comment:
Since all methods are about action with the same entity, `Account`, it might be an indicator that all these belong to `Account` itself:

```
$account->debit();
$account->credit();
$account->balance();

```

Reply:
You means account methods with signature: 

```php
public function debit(CalculatorInterface $calculator): MoneyBag 
```

?

Usage will be:

```php
$account->debit($calculator);
```

or 

```php
$calculator->debit($account);
```

Seems, it's equal... Does it make sense to add methods `debit`/`credit`/`balance` to `Account` entity?

Reply:
Decided to keep current solution.